app/helpers/ct_table_for/application_helper.rb

```diff
@@ -176,8 +176,8 @@ def table_for_actions(record, options: {} )
       html << %Q{<td data-link-enabled="false">}
```

Comment:
this method is extra large, can we extract parts into smaller methods to make it more readable?